File: app/src/main/java/no/nav/tilbakemeldingsmottak/consumer/saf/journalpost/DataJournalpost.java
package no.nav.tilbakemeldingsmottak.consumer.saf.journalpost;

import java.io.Serializable;

public class DataJournalpost implements Serializable {
	private SafJournalpostTo journalpost;

	public SafJournalpostTo getJournalpost() {
		return journalpost;
	}

	public void setJournalpost(SafJournalpostTo journalpost) {
		this.journalpost = journalpost;
	}
}

File: app/src/main/kotlin/no/nav/tilbakemeldingsmottak/consumer/saf/journalpost/DataJournalpost.kt
package no.nav.tilbakemeldingsmottak.consumer.saf.journalpost

import java.io.Serializable
import no.nav.tilbakemeldingsmottak.saf.generated.hentjournalpost.Journalpost


class DataJournalpost : Serializable {
    var journalpost: Journalpost? = null
}